### PR TITLE
LPS-197463 Show the hidden objectDefiniton2 when creating the relationship in model builder

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPS-197463

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @danvb112 :smile: 
Since you are a member of the BPM team, you are expected to assign someone from the team as the reviewer.
Assigning the reviewer is a way to organize our line and make the review process agile.
If you don't know who to assign, please talk with @gabrielwas.

**Comment by danvb112** [comment]
Hey @marcelabc, sorry I forgot to assignee a person, done! Thanks for remember me! 😁

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx [hunk 1]
```diff
@@ -101,27 +101,57 @@ export default function EditObjectFolder({
...
 		const payload = await getUpdatedModelBuilderStructurePayload(
 			baseResourceURL,
 			selectedObjectFolder.name
 		);
 
+		if (
+			newObjectRelationship.objectDefinitionId1 !==
+			newObjectRelationship.objectDefinitionId2
+		) {
+			const objectDefiniton2 = nodes.find(
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx:116]: Hey, I believe you meant `objectDefinition2`

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx [hunk 2]
```diff
@@ -101,27 +101,57 @@ export default function EditObjectFolder({
...
+		newObjectRelationship: ObjectRelationship
 	) => {
 		const payload = await getUpdatedModelBuilderStructurePayload(
 			baseResourceURL,
 			selectedObjectFolder.name
 		);
 
+		if (
+			newObjectRelationship.objectDefinitionId1 !==
+			newObjectRelationship.objectDefinitionId2
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx:114]: When the choice regarding the use of Object Definition ID or Object Definitions External Reference Code does not harm the code, I believe we should choose to use the external reference code.

**Review by igor-franca — CHANGES_REQUESTED** [review]
As Daniel started his vacation, I will close this PR and open others with the suggested changes.

**Comment by igor-franca** [comment]
resent [here](https://github.com/liferay-objects/liferay-portal/pull/3039)

**Pull request closed without merging**